purchase_requisition_for_everybody/view/purchase_requisition_view.xml

```diff
@@ -59,5 +59,16 @@
         action="purchase_requisition.action_purchase_requisition"
         />
 
+    <record model="ir.ui.view" id="res_users_hide_requistion_group">
+        <field name="name">res.users.hide.requistion.group</field>
+        <field name="model">res.users</field>
+        <field name="inherit_id" ref="base.user_groups_view"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='sel_groups_54_55']" position="attributes">
```

Comment:
@nhomar in the xml of the original view I found [this](https://github.com/vauxoo/odoo/blob/8.0/openerp/addons/mabase/res/res_users_view.xml#L238-L245) and I check for another view that inherit from the principal but I dont found any.

I search for the xml id of the group as you say but when I added the next error was thow
```
Error details:
Element '<xpath expr="//field[@name='purchase_requisition.group_purchase_requisition_manager']">' cannot be located in parent view
```

I dont now how to manage this.Can you help me please?
Regards